static/js/lib/learning_resources.js

```diff
@@ -27,24 +34,131 @@ export const availabilityLabel = (availability: ?string) => {
   }
 }
 
+export const parseDateFilter = (filter: string) => {
+  const format = /(now)(\+)?(\d+)?([Md])?/
+  const match = format.exec(filter)
+  if (match) {
+    let dt = moment()
+    if (match[3] && match[4]) {
+      dt = dt.add(match[3], match[4] === "d" ? "days" : "months")
+    }
+    return dt
+  }
+}
+
+export const inDateRanges = (run: CourseRun, availabilities: Array<string>) => {
+  let inRange = false
+  availabilities.forEach(availability => {
+    if (AVAILABILITY_MAPPING[availability]) {
+      const from = parseDateFilter(
```

Comment:
this will always be `moment.now()` I think, so the `parseDateFilter` function doesn't need to handle this case

Reply:
`from` will be `null` for `Available Now`

Reply:
gotcha, in that case, if it's `moment.now()` for all other cases it might be easier to just put that logic right here, like `from = availability === AVAILABLE_NOW ? null : moment.now()`